Scripts/azure_stack_provision.ps1

```diff
@@ -14,19 +12,18 @@ param (
 	[string]$sql_password
 )
 
+#####Server Name############
+$sql_server = "db-regression-$lansa_version" #only accepting lower case
 
-
+#######################Retriving Azure Resources using Tags ##################### 
+$db = "test"
```

Comment:
For Azure sql server there is always going to be more than 1 db when you query for the db in that server. Apart from the ones we create, there will be a default db called "master". The reason I am using $(db)